.gitignore

```diff
@@ -38,4 +39,6 @@ obj/
 *.idb
 *.opendb
 WSATestCertificate.pfx
-WSATestCertificate.pfx.meta
\ No newline at end of file
+WSATestCertificate.pfx.meta
+
+!*SRMesh.obj
```

Comment:
What's with the custom obj ref?

Should we allow obj extensions in general? (Pretty much the same as .fbx)

Reply:
.obj is also used for some Visual Studio generated files, but I think we might catch all those with the obj/ folder ignore. It's probably safe to remove all *.obj ignores, negating the need for this specific one

Reply:
Agreed. I've never had issues with VS obj objects in my projects with the Obj/ folder ignore.